src/spacegame3/SpaceGame.java

```diff
@@ -80,29 +94,45 @@ public void giveSceneTo(SizableScene scene){
         stage.setScene(scene);
     }
 
+    /*
+        *main method - runs + launches game
+    */
     public static void main(String[] args) {
         launch(args);
     }
 
+    //getter for game started
     public boolean gameStarted() {
         return gameStarted;
     }
 
-
+    /*
+        *sets the planet screen to have landed on the parameter (a CelestialBody object)
+        *returns planet screen
+    */
     public SizableScene getPlanetScreen(CelestialBody planet) {
         planetScreen.setLandedOn(planet);
         return planetScreen;
     }
 
+    /*
+        *saves the current scene
+        *transfers to the start screen
+    */
```

Comment:
use /** */ for methods (just repeating for each one)
also just change the word saves for stores
for me saves has a more long term sense than what is done here